### ISSUE TITLE
NVDA doesn't read paragraph correctly in open/libre/br office

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2299]
**Reported by alente on 2012-05-05 22:03**
In open/libre/br office, when we press ctrl+down arrow, the NVDA reads only the first line of paragraph.

If, after press ctrl+down arrow, we press ctrl+shift+down arrow, the NVDA reads all paragraph.

Thanks

Alexandre

**Comment by leonardder** [comment]
Can't reproduce in IA2 based Libre Office. This issue dated from the time we were still using JAB. Closing.
